### PR TITLE
docs: use one function for all tracking

### PR DESCRIPTION
**Pull request by Zaryafaraj**

### CODE REVIEW

**Comment by Zaryafaraj** [comment]
@ryandotsmith PTAL

File: docs/core/get-started/install.partial.html [hunk 1]
```diff
@@ -132,7 +132,7 @@ <h3 data-no-anchor="">Linux</h3>
     <span class="close"><img src="/docs/images/close.png" class="close-icon"> </span>
     <div class="form-container">
       <div class="modal-title ">Stay up to date with Chain Core</div>
-      <p class="modal-subtitle">We're constanty developing new features and we want you to be the first to try them. Enter your email
+      <p class="modal-subtitle">We're constantly developing new features and we want you to be the first to try them. Enter your email
```
Comment (kr) [inline docs/core/get-started/install.partial.html]: Is this change intentional here? It seems unrelated to the rest of the commit.

Comment (Zaryafaraj) [inline docs/core/get-started/install.partial.html]: took this out for another PR. thanks!
